### PR TITLE
Updates for docker, .travisyml, and numpy 1.16

### PR DESCRIPTION
**Pull request by canyon289**

### CODE REVIEW

**Comment by canyon289** [comment]
Waiting for this issue to be resolved
https://github.com/pymc-devs/pymc3/issues/3340

**Comment by ahartikainen** [comment]
Add this to line 39 (after pip upgrade) in `create_testenv.sh`

    pip --no-cache-dir install numpy<1.16

or 

    pip --no-cache-dir install numpy==1.15.*

**Comment by canyon289** [comment]
Sounds good. I'm revamping the dockerfile and .travis.yml file. I'll do this all together

**Comment by nouiz** [comment]
We just released Theano 1.0.4 that added support for numpy 1.16 on the CPU. We didn't test on the GPU.

File: scripts/Dockerfile [hunk 1]
```diff
@@ -1,37 +1,46 @@
 FROM conda/miniconda3
 
-
-MAINTAINER Ravin Kumar
+LABEL maintainer="SvenDowideit@home.org.au"
```
[the remark this answers is not in the document]
Comment (canyon289) [reply scripts/Dockerfile]: I took the docker documentation too literally

File: .travis.yml [hunk 2]
```diff
@@ -47,23 +47,22 @@ install:
...
   # Tests command is default in dockerfile cmd
-  - if [[ $NAME == UNIT ]]; then docker run  --mount type=bind,source="$(pwd)",target=/opt/arviz --name arviz_test arviz:latest /bin/bash -c \
-                                              "source /root/.bashrc && NUMBA_DISABLE_JIT=1 pytest -v arviz/tests/ --cov=arviz/";
+  - if [[ $NAME == UNIT ]]; then docker run  --mount type=bind,source="$(pwd)",target=/opt/arviz --name arviz_test arviz:latest bash -c \
+                                              "NUMBA_DISABLE_JIT=1 pytest -v arviz/tests/ --cov=arviz/";
     fi
 
-  - if [[ $NAME == LINT ]]; then docker run arviz:latest /bin/bash -c \
-                            "source /root/.bashrc && ./scripts/lint.sh";
+  - if [[ $NAME == LINT ]]; then docker run arviz:latest /bin/bash -c "./scripts/lint.sh";
```
Comment (ahartikainen) [inline .travis.yml]: /bin/bash -> bash

File: arviz/stats/stats.py [hunk 3]
```diff
@@ -464,7 +464,7 @@ def psislw(log_weights, reff=1.0):
         xcutoff = max(x[x_sort_ind[cutoff_ind]], cutoffmin)
 
         expxcutoff = np.exp(xcutoff)
-        tailinds, = np.where(x > xcutoff)
+        tailinds, *_ = np.where(x > xcutoff)
```
Comment (ahartikainen) [inline arviz/stats/stats.py]: np.flatnonzero?

File: scripts/Dockerfile [hunk 4]
```diff
@@ -1,37 +1,46 @@
...
+# Change behavior of create_test.sh script
 ENV DOCKER_BUILD=true
 
 # For Sphinx documentation builds
 ENV LC_ALL=C.UTF-8
 ENV LANG=C.UTF-8
 
-RUN apt-get update && apt-get install -y git build-essential pandoc \
+# Update container
+RUN apt-get update && apt-get install -y git build-essential pandoc vim \
```
Comment (ahartikainen) [inline scripts/Dockerfile]: Thanks for vim

File: .travis.yml (hunk 2 above)
Comment (canyon289) [reply .travis.yml]: Yes need to do

File: arviz/stats/stats.py (hunk 3 above)
Comment (canyon289) [reply arviz/stats/stats.py]: This one I was confused about. Should I just add pylint or was it giving the correct message?

And in regards to your comments how does `np.flatnonzero` help here? Looked at the docs but couldn't understand
https://docs.scipy.org/doc/numpy/reference/generated/numpy.flatnonzero.html

Comment (canyon289) [reply arviz/stats/stats.py]: Changed back to original statement

Comment (ahartikainen) [reply arviz/stats/stats.py]: Your solution looks good

File: arviz/stats/stats.py [hunk 5]
```diff
@@ -464,7 +464,7 @@ def psislw(log_weights, reff=1.0):
         xcutoff = max(x[x_sort_ind[cutoff_ind]], cutoffmin)
 
         expxcutoff = np.exp(xcutoff)
-        tailinds, = np.where(x > xcutoff)
+        tailinds, = np.where(x > xcutoff)  # pylint: disable=unbalanced-tuple-unpacking
```
Comment (canyon289) [inline arviz/stats/stats.py]: @ahartikainen, is this correct now? Just want to double check

Comment (ahartikainen) [reply arviz/stats/stats.py]: Yes, it will return tuple with one item.

Comment (canyon289) [reply arviz/stats/stats.py]: Great merging

**Comment by ahartikainen** [comment]
LGTM

**Pull request merged**